package-lock.json

```diff
@@ -350,12 +360,12 @@
             "dev": true
         },
         "node_modules/@node-oauth/oauth2-server": {
-            "version": "4.3.0",
-            "resolved": "https://registry.npmjs.org/@node-oauth/oauth2-server/-/oauth2-server-4.3.0.tgz",
-            "integrity": "sha512-QxSWMPwiEFwZczFbZRJjmzsRT9rgb3FhBw/rZlFe9+ZrOWEc6r7mFCcvX14ekxLHSxAxZEYIgzUtE4yHfKiTxg==",
+            "version": "4.3.2",
+            "resolved": "https://registry.npmjs.org/@node-oauth/oauth2-server/-/oauth2-server-4.3.2.tgz",
+            "integrity": "sha512-hZDXOWkhZF1edffMf+NERjnRBj839N9YzYkKNZ0/+E2Q2AvvIZOtuvqVkHZYIiYEolv1E3Fr22hUb8nlxiTArQ==",
```

Comment:
Still not resolved, This dependency is coming from binding-http package and it is unnecessarily updated...